packages/odyssey-storybook/src/components/Field/Field.mdx

```diff
@@ -1,43 +1,150 @@
+import { Canvas, Story, ArgsTable } from "@storybook/addon-docs";
+
 # Field
 
-These captions help make forms more accessible by providing context to the user. They can be used with all Odyssey inputs.
+Field provides all of the supporting context required by inputs, including labels, hints, and errors.
+
+This component is also responsible for managing the appropriate application of `aria-` attributes and `id` associations between Inputs and their associated elements.
+
+[View source on GitHub](https://github.com/okta/odyssey/tree/master/packages/odyssey-react/src/components/Field)
 
 ## Anatomy
```

Comment:
can we include the prop table up top here for engineers looking for the API? It appears to be missing here in the docs page entirely.

Incidentally, you can also fix the broken prop tables for Field in this PR which would be great. [We're using a new approach described here.](https://okta.slack.com/archives/G01JS0FE32P/p1638891117039900?thread_ts=1638890977.039800&cid=G01JS0FE32P)

Reply:
The prop table is just below, but wasn't populating before. Added a `Field.docgen` - working great now!